src/symbols.js

```diff
@@ -739,7 +740,8 @@ for (let i = 0; i < letters.length; i++) {
 // but they are not actually in the font, nor are they supported by the
 // Unicode accent mechanism, so they fall back to Times font and look ugly.
 // TODO(edemaine): Fix this.
-export const extraLatin = "ÇÐÞçþ";
+export const extraLatin = "ÐÞþ";
+
```

Comment:
You'll need to add the cedilla c's to the math symbol table similar to the "ð" below, unless we add a math version of the accent.

Reply:
Somehow I thought we merged this already, but it looks like the above is still an outstanding issue.  We have two options:
1. Add a `\mathcedilla` or other nonstandard operator for a math version of the cedilla accent, and add support for `ç` etc. in math mode.
2. Add the following lines next to the similar lines for "ð":
```js
 defineSymbol(text, main, mathord, "ç", "ç");
 defineSymbol(text, main, mathord, "Ç", "Ç");
```

The former approach will give the proper glyphs; the latter approach will maintain backward compatibility by forcing çÇ to still fall back to system fonts.